### PR TITLE
Added animation 

### PR DESCRIPTION
**Pull request by ananya**
issue #12

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
Please provide live link (gh-pages).

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (Anupam-dagar) [inline _config.yml]: remove this

**Comment by ananya** [comment]
https://ananya-agrawal.github.io/Girls-who-code/.

**Comment by Anupam-dagar** [comment]
@pradeepgangwar @aswinzz we have 2 logos, time to make a choice, tell your preference in "core" strean on zulip
coming to animation @ananya-agrawal it looks awesome

**Comment by Anupam-dagar** [comment]
is this meant to be like this or is it a responsiveness issue (by foss circle)? 
screenshot from 2018-01-12 23-48-43

**Comment by pradeepgangwar** [comment]
@ananya-agrawal I deleted jekyll theme file. We really don't need that. Gh-pages should work without that too.

**Comment by ananya** [comment]
@Anupam-dagar  sir it is a responsive one and linked to the official website of foss

**Comment by Anupam-dagar** [comment]
Don't call anybody sir on GitHub :smile: 
make it responsive

**Review by Anupam-dagar — APPROVED** [review]

**Pull request merged**